src/main/java/core/basesyntax/service/StoreServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public class StoreServiceImpl implements StoreService {
```

Comment:
```suggestion
public class StoreServiceImpl implements ReportService {
```
Maybe report service will be better?

Reply:
fixed